pystats_docs.md

```diff
@@ -0,0 +1,81 @@
+# pystats documentation
+
+This describes the meaning of the various sections and fields in the pystats output.
+
+## Execution counts
+
+This is the counts of how many times each Tier 1 instruction is executed.
+
+The "miss ratio" column shows the percentage of times the instruction executed that it deoptimized.
+
+## Pair counts
+
+Shows the Top 100 most frequently occurring pairs of instructions as executed.
```

Comment:
Similar question — does a specialized op that deoptimizes form a “pair” with the subsequently executed unspecialized op?

Reply:
No.